server/src/models/Salad.js

```diff
@@ -0,0 +1,39 @@
+const Model = require("./Model.js")
+
+
+class Salad extends Model{
+    static get tableName() {
+        return "salads"
+    }
+
+    static get relationMappings() {
+        const { User } = require("./index.js")
+
+        return {
+            user: {
+                relation: Model.BelongsToOneRelation,
+                modelClass: User,
+                join: {
+                    from: "salads.userId", 
+                    to: "users.id"
+                }
+            }
+        }
+    }
+
+    static get jsonSchema() {
+        return {
+            type: 'object',
+            required: ["name"],
+            properties: {
+                name: {type: "string"},
+                description: {type: "string"}
```

Comment:
info in braces should be space padded - `{ type: "string" }`
https://github.com/airbnb/javascript#whitespace--in-braces